README.md

```diff
@@ -1,10 +1,45 @@
 # Fabulous-chocolate-factory
-Projeto para estudo
 
-Criar uma aplicação com feign -> retry -> trace id via request -> teste de contrato.
-Eventos internos -> trace id via evento -> subir kafka no docker -> salva eventos em banco de dados -> teste de container . RabbitMQ -> log -> teste de integração usando kafka embadded
-Configura uma esteira -> resilience 4J
-Colocar Doc swagger, configurar docker para subir todas as aplicações com docker compose
+
+# About this project
+
+This project is a study case of some spring and kotlin libs
+
+# Getting Started
+(TODO)
+
+### TODO LIST
+
+- [ ] Chocolate Factory
+    - [ ] Inventory control - Gets ingredients from the store
+        - [ ] Feing 
+        - [ ] Retry
+        - [ ] Trace id
+        - [ ] Contract tests
+    - [ ] Produce chocolate
+        - [ ] Trace id
+        - [ ] 
```

Comment:
Maybe you don't need this check point, because has no text

Reply:
My bad hehehehe :rocket: